### ISSUE TITLE
torch.nn.ConvTranspose2d

### ISSUE THREAD

**Issue by ytgui** [bosskwei/temp#59]
https://zhuanlan.zhihu.com/p/48501100
+ 卷积是卷积的逆过程，又称作转置卷积。最大的区别在于反卷积过程是有参数要进行学习的（类似卷积过程），理论是反卷积可以实现UnPooling和unSampling，只要卷积核的参数设置的合理。
+ 反卷积的操作只是恢复了矩阵 X 的尺寸大小，并不能恢复 X 的每个元素值

torch.nn.LeakyReLU
+ Leaky version of a Rectified Linear Unit.
+ Leaky ReLUs allow a small, non-zero gradient when the unit is not active.

Parametric ReLUs take this idea further by making the coefficient of leakage into a parameter that is learned along with the other neural network parameters.

![image](https://user-images.githubusercontent.com/12762888/55133815-dd6b8700-5161-11e9-93f5-645c5771a2b6.png)
